recipes/gw-sklearn-xarray/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sgw-sklearn-xarray" %}
```

Comment:
Just a last clarification before we merge. The title of the PR says "gw-sklearn-xarray", while here we have "sgw-sklearn-xarray". Can you confirm name is set correctly here or otherwise make any needed correction?